packages/arb-token-bridge-ui/src/components/TransferPanel/Routes/Route.tsx

```diff
@@ -96,10 +98,290 @@ function getBadges(badgeTypes: BadgeType | BadgeType[]) {
   if (Array.isArray(badgeTypes)) {
     return badgeTypes.map(getBadgeFromBadgeType)
   }
-
   return getBadgeFromBadgeType(badgeTypes)
 }
 
+// Route Amount Component
+type RouteAmountProps = {
+  destinationAddress?: string
+  amountReceived: string
+  token: Token | NativeCurrency
+  showUsdValueForReceivedToken: boolean
+  ethToUSD: (amount: number) => number
+  isBatchTransferSupported: boolean
+  amount2?: string
+  childNativeCurrency: Token | NativeCurrency
+  compactMode: boolean
+}
+
+const RouteAmount = ({
+  destinationAddress,
+  amountReceived,
+  token,
+  showUsdValueForReceivedToken,
+  ethToUSD,
+  isBatchTransferSupported,
+  amount2,
+  childNativeCurrency,
+  compactMode
+}: RouteAmountProps) => (
+  <div className="flex flex-col md:min-w-36">
+    {!compactMode && (
+      <span className="flex gap-1">
+        {destinationAddress ? (
+          <Tooltip content={destinationAddress}>
+            {shortenAddress(destinationAddress)}
+          </Tooltip>
+        ) : (
+          'You'
+        )}{' '}
+        will receive:
+      </span>
+    )}
+    <div
+      className={twMerge('flex flex-col text-lg', compactMode && 'text-base')}
+    >
+      <div className="flex flex-row items-center gap-1">
+        <TokenLogo
+          srcOverride={'logoURI' in token ? token.logoURI : null}
+          fallback={
+            <div className="h-5 w-5 min-w-5 rounded-full bg-gray-dark" />
+          }
+        />
+        {formatAmount(Number(amountReceived))} {token.symbol}
+        <div className="text-sm">
+          {showUsdValueForReceivedToken && (
+            <div
+              className={twMerge(
+                'tabular-nums opacity-80',
+                compactMode ? 'text-xs' : 'text-sm'
+              )}
+            >
+              ({formatUSD(ethToUSD(Number(amountReceived)))})
```

Comment:
`amountReceived` is always called with `formatUSD` and `ethToUSD`. We might as well just send the formated amount, and remove `ethToUSD` from the props